### PR TITLE
modify native staking message

### PR DESCRIPTION
**Pull request by koseoyoung**
add stakeAdd stakeAgain message

### CODE REVIEW

File: proto/types/action.proto [hunk 1]
```diff
@@ -60,29 +60,56 @@ message Bucket {
...
 
-// move the bucket to vote for another candidate
+// move the bucket for another candidate or transfer the ownership of bucket to another voter  
 message StakeMove {
-  string candidateName = 1;
-  uint64 bucketIndex = 2;
-  bytes payload = 3;
+  StakeMoveType type = 1;
+  string name = 2;
+  uint64 bucketIndex = 3;
```
Comment (dustinxie) [inline proto/types/action.proto]: make bucketIndex = 2; consistent with other definition

File: proto/types/action.proto [hunk 2]
```diff
@@ -60,29 +60,56 @@ message Bucket {
...
+
+// unstake or withdraw
 message StakeUpdate {
+  StakeUpdateType type = 1;
+  uint64 bucketIndex = 2;
+  bytes payload = 3;
+}
+
+// add the amount of bucket 
+message StakeAddDeposit {
```
Comment (dustinxie) [inline proto/types/action.proto:83]: combine AddDeposit and Again?

**Review by dustinxie — COMMENTED** [review]
2 comments

File: proto/types/action.proto (hunk 1 above)
Comment (koseoyoung) [reply proto/types/action.proto]: yes, fixed it on latest commit

**Review by dustinxie — APPROVED** [review]

**Pull request merged**